File: corenlp.py
import requests,json

# The variables
key ="api_key"
secret ="api_secret"
url ="https://corenlp.ai"
text ="the fox jumped over the lazy dog"

# Issue the request
response = requests.post(url, data="the fox jumped over the lazy dog")
annotated = json.loads(response.text)

# Print, e.g., the relation triples in the sentence
triples = map ( lambda x: "(%s; %s; %s)" % (x['subject'], x['relation'], x['object']), annotated['sentences'][0]['openie'] )

for triple in triples:
   print(triple)

File: corenlp.js
var request = require('request');

// The parameters of the POST request
var params = {
   url: 'https://corenlp.ai',
   auth: {
      user: 'api_key',
      password: 'api_secret'
   },
   form: 'The brown fox jumped over the lazy dog'
}

// Issue the POST request
request.post(params,
   function (error, response, annotated_text) {
      if (!error && response.statusCode == 200) {
         // Process the annotated text
         console.log(annotated_text)
      }
   }
);
